interface/src/app/utils.js

```diff
@@ -17,4 +17,24 @@ angular.module('adage.utils', [])
 
 // Number of digits in activity value shown on frontend:
 .constant('ActivityDigits', 5)
+
+// Service of global machine learning model, which is shared by multiple
+// modules.
```

Comment:
I understand why you've put this here, but I think my preference would be to reinforce the component architecture by defining this service in a file within the `src/app/mlmodel` directory and then list that file as a *dependency* for `adage.utils` above.